[Dotenv] Add `--override` option (#58540)
symfony/symfony: src/Symfony/Component/Dotenv/CHANGELOG.md

CHANGELOG
=========

7.2
---

> * Introduce an --override option to specify whether to overwrite existing .env.local.php file

Reviewer: ```suggestion
 * Add support for the `--override` option to overwrite existing `.env.local.php` file
```

Author: done. thank you @alexandre-daubois
